cfg/posix.cfg

```diff
@@ -5926,6 +5932,7 @@ The function 'mktemp' is considered to be dangerous due to race conditions and s
   <!-- http://pubs.opengroup.org/onlinepubs/000095399/functions/sigismember.html -->
   <!-- int sigismember(const sigset_t *set, int signum);-->
   <function name="sigismember">
+    <pure/>
```

Comment:
This is not `pure` as the object pointed to might change during run-time.

Reply:
`strlen()` is designated as a pure function here: https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html

Reply:
Wrong comment.